frontend/packages/dev-console/src/plugin.tsx

```diff
@@ -294,6 +296,17 @@ const plugin: Plugin<ConsumedExtensions> = [
         ).then((m) => m.default),
     },
   },
+  {
+    type: 'Overview/Resource',
+    properties: {
+      name: 'Resources',
+      key: 'configurations',
+      loader: () =>
+        import(
+          './components/monitoring/overview/MonitoringTab' /* webpackChunkName: "knative-overview" */
+        ).then((m) => m.default),
+    },
+  },
```

Comment:
is this being used?

Reply:
Now enhanced to add new tabs from plugins